state/keys.go

```diff
@@ -61,6 +61,15 @@ func (k Keys) ChunkSizes() ([]uint16, bool) {
 	return chunks, true
 }
 
+// Strip returns the database keys of k
+func (k Keys) Strip() []string {
```

Comment:
Could we rename this to either `Keys()`, `KeysWithoutPermissions()`, or `WithoutPermissions()` ?

Reply:
Will rename to `WithoutPermissions()`

Reply:
Done